docs/separate-benchmark-module.md

```diff
@@ -0,0 +1,102 @@
+# Creating Separate Module for Benchmarks
+
+This guide will walk you through the process of creating a separate module for benchmarks in your Kotlin project. 
+Creating a separate module for benchmarks is especially beneficial when you are integrating benchmarks into an existing project. 
+Here are a few advantages of doing so:
+
+1. **Organization**: It helps maintain a clean and organized project structure. Segregating benchmarks from the main code makes it easier to navigate and locate specific code segments.
+
+2. **Isolation**: Separating benchmarks ensures that the benchmarking code does not interfere with your main code or test code. This isolation guarantees accurate measurements without unintentional side effects.
```

Comment:
But that could be also achieved using a separate source set, not a module.
It's unclear why there should be issues with the measurements accuracy when benchmarks are placed along with main sources.

Reply:
And it is the source set.

Reply:
> But that could be also achieved using a separate source set, not a module.

https://github.com/Kotlin/kotlinx-benchmark/pull/175#discussion_r1437011229

> t's unclear why there should be issues with the measurements accuracy when benchmarks are placed along with main sources.

Agree. I will remove this point.